com.sap.cloud.lm.sl.cf.core/src/test/java/com/sap/cloud/lm/sl/cf/core/cf/v2/ConfigurationEntriesCloudModelBuilderTest.java

```diff
@@ -74,7 +80,7 @@ public void parseDeploymentDescriptor() {
 
     @Test
     public void testBuild() {
-        ConfigurationEntriesCloudModelBuilder builder = new ConfigurationEntriesCloudModelBuilder(orgName, spaceName, "");
+        ConfigurationEntriesCloudModelBuilder builder = new ConfigurationEntriesCloudModelBuilder(orgName, spaceName, "", namespace);
```

Comment:
Can you refactor this test using Parameterized and MethodSource annotations from junit5?